### PR TITLE
Entrypoint Function

### PR DESCRIPTION
**Pull request by typ3c4t**

### CODE REVIEW

File: transpiler/src/zero_generator.rs [hunk 1]
```diff
@@ -297,18 +302,14 @@ impl Transpiler {
...
-    };
-
-    todo!()
+//Return a list of Generalized System Zero instructions from a Vec of Yul expressions
+pub fn transpile_program(expressions: Vec<Expr>) -> Vec<GeneralInstruction> {
+    let mut transpiler = Transpiler::new();
+    for expression in &expressions {
+        transpiler.transpile_op(expression);
+    }
+    transpiler.instructions
```
Comment (typ3c4t) [inline transpiler/src/zero_generator.rs:314]: the primary change

File: transpiler/src/zero_generator.rs [hunk 2]
```diff
@@ -135,6 +133,13 @@ struct Branch {
 }*/
 
 impl Transpiler {
+    fn new() -> Self {
```
Comment (typ3c4t) [inline transpiler/src/zero_generator.rs:136]: `new()` method for `Transpiler`

File: transpiler/src/zero_generator.rs [hunk 3]
```diff
@@ -135,6 +133,13 @@ struct Branch {
...
+        Transpiler {
+            instructions: Vec::new(),
+            current_stack_frame: StackFrame::new(),
+            previous_stack_frames: Vec::new(),
+        }
+    }
```
Comment (npwardberkeley) [inline transpiler/src/zero_generator.rs:140]: nit: newline here?

**Review by npwardberkeley — APPROVED** [review]
Looks great!

**Pull request merged**